back/.github/workflows/phpstan-weekly.yml

```
@@ -0,0 +1,216 @@
+        required: false
+        default: true
+        type: boolean
+
+jobs:
+  phpstan-debt-tracking:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      issues: write
+      pull-requests: write
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)